### DISCUSSION TITLE
TODO.md update

### DISCUSSION THREAD

**Discussion by THEkatinamicrowave** [CodenameCrew/CodenameEngine#488] [Q&A]
Correct me if I'm wrong, @NeeEoo @NexIsDumb but I'm fairly certain that these are implemented
![image](https://github.com/user-attachments/assets/b18e271e-9628-4efa-9f48-a0964360bf92)

**Comment by NeeEoo** [comment]
Its not

**Reply by THEkatinamicrowave** [reply]
Ok. In that case, what do those entail?

**Reply by r6915ee** [reply]
> Its not

Is the notetype feature just incomplete? I've seen how most of it is by script, but otherwise it is completely functional.
